test/norm_flows.jl

```diff
@@ -0,0 +1,24 @@
+using Test
+using Bijectors, ForwardDiff, LinearAlgebra
+
+@testset "planar flows" begin
+    for i in 1:10
+        flow = PlanarLayer(10)
+        z = randn(10,100)
+        forward_diff = log(abs(det(ForwardDiff.jacobian(t -> transform(flow, t), z))))
+        our_method = sum(forward(flow, z).logabsdetjacob)
+        @test our_method ≈ forward_diff
+        # @test inv(flow, transform(flow, z)) ≈ z
```

Comment:
@sharanry Why do we comment this out?

Reply:
When we tried with dummy alpha and beta, we were getting the exact result. However, with randomized alpha, beta and data, the inverse is not perfect.

Reply:
I understand the inverse is not perfect but we should still get close enough results.
Can you also add a test with dummy alpha and beta and see if it passes?